.github/ISSUE_TEMPLATE.md

```diff
@@ -1,14 +1,29 @@
-* xeus-cookiecutter version:
+# Checklist
+Before submitting an issue please confirm you have checked the follow
+- [ ] This issue is not a duplicate of one already open
+- [ ] The title of the issue is clear
+
+# System information
+* xeus-cpp version:
 * Operating System:
+* CPU architechture:
```

Comment:
Typo: architechture => architecture
I would add examples of architectures in parenthesis, as beginners may not understand what this means